### ISSUE TITLE
false positive on setuptools.pkg_resources.get_distribution return value (E1103)

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#36]
Originally reported by: **Jace Browning (BitBucket: [jacebrowning](http://bitbucket.org/jacebrowning), GitHub: @jacebrowning?)**

----------------------------------------

For the following code:

```
#!python

from pkg_resources import get_distribution
print(get_distribution('setuptools').version)
```

I get:

```
#!bash

E1103: 2,6: Instance of 'str' has no 'version' member (but some types could not be inferred)
```

Expected:

No warning should occur because the return value of `get_distribution` does, in fact, have a `version` member.

Version:

```
#!bash

pylint 0.28.0,
astng 0.24.3, common 0.59.1
Python 2.7.2 (v2.7.2:8527427914a2, Jun 11 2011, 15:22:34)
[GCC 4.2.1 (Apple Inc. build 5666) (dot 3)]
```

----------------------------------------
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/36

**Comment by PCManticore** [comment]
Fixed in astroid's master branch. https://github.com/PyCQA/astroid/commit/a944eac910b9422adf647e9bd966f1339142562f?diff=unified
